src/app/components/BibPage/BibPage.jsx

```diff
@@ -107,17 +107,21 @@ class ItemPage extends React.Component {
       { label: 'Number available', field: 'numAvailable' },
       { label: 'Number of items', field: 'numItems' },
       { label: 'Shelf Mark', field: 'shelfMark' },
-    ]
+    ];
+    let shortenItems = true;
+    if (this.props.location.pathname.indexOf('all') === -1) {
```

Comment:
Same component for both routes but if the url does not contain `all`, then it will display all items. Did this so that can be the default when JS is turned off.